frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileType.java

```diff
@@ -3,9 +3,9 @@
 import javax.swing.Icon;
 
 import com.intellij.openapi.fileTypes.LanguageFileType;
-import icons.MacheteIcons;
```

Comment:
@mkondratek wasn't there a requirement for these icons to reside in `icons` package? if so, it should be documented in package-info.java

@MaciejG604 are the icons still rendered correctly everywhere in the plugin?

Reply:
Nvm, see `frontend/icons/src/main/java/icons/MacheteIcons.java` below